app/src/main/java/com/simplecity/amp_library/ui/detail/genre/GenreDetailPresenter.kt

```diff
@@ -84,7 +85,7 @@ class GenreDetailPresenter constructor(private val genre: Genre) : Presenter<Gen
             .delay(if (currentSlideShowAlbum == null) 0L else 8L, TimeUnit.SECONDS)
 
         addDisposable(Observable
-            .combineLatest(a, b, BiFunction { albums: List<Album>, aLong: Long -> albums })
+            .combineLatest(a, b, BiFunction { albums: List<Album>, _: Long -> albums })
```

Comment:
There's nothing wrong with this change, but it probably should be part of a separate commit, since it has nothing to do with `MusicUtils`